lib/org/openpsa/invoices/invoice/pdf.php

```diff
@@ -30,7 +30,7 @@ public function get_attachment(bool $autocreate = false) : ?midcom_db_attachment
         if (!$autocreate) {
             return null;
         }
-        return $this->render_and_attach();
+        return $this->render_and_attach('bill');
```

Comment:
I'd say let's make this parameter optional, and if it's empty we get the same behavior as now